src/smartpeak/source/core/RawDataProcessor.cpp

```diff
@@ -2048,9 +2184,24 @@ namespace SmartPeak
     LOGD << "END calculateMDVs";
   }
 
+  ParameterSet IsotopicCorrections::getParameterSchema() const
+  {
+    std::map<std::string, std::vector<std::map<std::string, std::string>>> param_struct({
+    {"IsotopicCorrections", {
+      {
+        {"name", "correction_matrix_agent"},
+        {"type", "string"},
+        {"value", "TBDMS"},
+        {"description", ""},
```

Comment:
"The correction matrix corresponding to the derivatization agent used when processing the samples for LC-MS/MS or GC-MS."